indent/ps1.vim

```diff
@@ -1,20 +1,87 @@
 " Vim indent file
-" Language:           Windows PowerShell
-" Maintainer:         Peter Provost <peter@provost.org>
-" Version:            2.10
-" Project Repository: https://github.com/PProvost/vim-ps1
-" Vim Script Page:    http://www.vim.org/scripts/script.php?script_id=1327"
+" Language:       Windows PowerShell
+" Maintainer:
+"   Lior Elia - http://blogs.msdn.com/lior
+"   Crane Jin <crane@cranejin.com>
+" Version:        2.0.1
+" Last Change:    2016 Aug 10
```

Comment:
Please leave hte "Project Repository" and the version and revision history isn't necessary. People can look at the git history and get a more complete history that doesn't require people manually update the history.